### PR TITLE
Feature/upgrade python django

### PR DESCRIPTION
**Pull request by deluge**
Heyho,

I´ve done the following things:

- add django 3.0.x support
- add django 3.1.x support
- add python 3.9.x support
- drop django 1.11.x support
- drop django 2.0.x support
- drop django 2.1.x support
- drop python 3.5.x support

because all of them are end of life. Additionally I moved all settings from setup.cfg to pyproject.toml, fixed some imports and made some mini mini improvements, if we can call it improvements ^^

If you´re fine with this PR you have to do some tasks:
1. merge PR
2. execute ```make release-tag```
3. go to https://github.com/deluge/django-didadata/releases/new choose the right tag and click "publish release"

### CODE REVIEW

File: didadata/__init__.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-__version__ = '0.4.0'
+__version__ = '0.5.0'
```
Comment (stephrdev) [inline didadata/__init__.py:1]: major release as we drop a lot of support

File: setup.py [hunk 2]
```diff
@@ -13,9 +13,9 @@
 
 
 prod_require = [
-    'Django>=1.11,<2.3',
-    'djangorestframework>=3.11.0,<3.12',
-    'django-filter>=2.2.0,<2.3',
+    'Django>=2.2,<3.2',
+    'djangorestframework>=3.12.1,<3.13',
```
Comment (stephrdev) [inline setup.py:17]: do we need to restrict upper version bounds?

**Review by stephrdev — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (deluge) [reply setup.py:17]: I don´t think so. I only followed the existing spelling ^^ I´ll remove it

File: didadata/__init__.py (hunk 1 above)
Comment (deluge) [reply didadata/__init__.py:1]: Okidoki

**Review by stephrdev — APPROVED** [review]

**Pull request merged**